akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala

```diff
@@ -37,9 +32,13 @@ object ClusterClientSpec extends MultiNodeConfig {
     akka.remote.log-remote-lifecycle-events = off
     akka.cluster.auto-down-unreachable-after = 0s
     akka.cluster.client.heartbeat-interval = 1s
-    akka.cluster.client.acceptable-heartbeat-pause = 3s
+    akka.cluster.client.acceptable-heartbeat-pause = 1s
```

Comment:
I'll revert the change here... I probably left it in by accident.

I've spent far too long writing these tests. I'd prefer to not have to focus on a new spec, although I take your point. Other than the above, I think the tests are otherwise preserved.